### PR TITLE
Chinese language screenshots

### PR DESCRIPTION
**Pull request by marcfishelquantil**
Please confirm these are satisfactory. There were a few screens, such as password reset, that I could not capture.

### CODE REVIEW

**Review by burtonquantil — CHANGES_REQUESTED** [review]
1) Your pull request has conflicts.
2) You're changing more than just the images.
3) You still refer to filenames with zh in them.  This shouldn't be necessary

File: en/docs/manifest.json [hunk 1]
```diff
@@ -117,6 +117,11 @@
                 "title": "Edit a Secret",
                 "path": "/docs/portal/secrets/editing-secrets.md"
               },
+            },
```
Comment (nibinqtl) [inline en/docs/manifest.json:120]: This seems to be a unmatched closing brace.

File: zh/docs/manifest.json [hunk 2]
```diff
@@ -117,6 +117,11 @@
                 "title": "修改保密信息",
                 "path": "/docs/portal/secrets/editing-secrets.md"
               },
+            },
```
Comment (nibinqtl) [inline zh/docs/manifest.json:120]: This seems to be a unmatched closing brace.

**Review by nibinqtl — CHANGES_REQUESTED** [review]

File: zh/docs/manifest.json (hunk 2 above)
Comment (burtonquantil) [reply zh/docs/manifest.json:120]: @nibinqtl  i expect Marc will close this pull request. It has a lot of problems. I talked to him about creating a new pull request starting with a clean copy of the repository.

**Comment by marcfishelquantil** [comment]
I am closing this pull request and will generate a new one when posting the latest files and English and Chinese screen shorts to a new branch.

**Pull request closed without merging**